utils/crypto/bls/secret.go

```diff
@@ -71,11 +71,3 @@ func Sign(sk *SecretKey, msg []byte) *Signature {
 func SignProofOfPossession(sk *SecretKey, msg []byte) *Signature {
 	return new(Signature).Sign(sk, msg, ciphersuiteProofOfPossession)
 }
-
-func DeserializeSecretKey(pkBytes []byte) *SecretKey {
-	return new(SecretKey).Deserialize(pkBytes)
-}
-
-func SerializeSecretKey(key *SecretKey) []byte {
-	return key.Serialize()
-}
```

Comment:
These functions were duplicates of `SecretKeyFromBytes` (without error handling + memory zeroing) and `SecretKeyToBytes`.